CONTRIBUTION.md

```diff
@@ -71,6 +71,9 @@ git tools.
   - Format your code and fix all the spelling/grammatical mistakes.
   - Limit the column length of the code and the comments within the code to 120 characters.
   - Error messages within the code should be limited to a single line.
+  - Document the details of your PR and put a link inside your PR comment
+  - Make sure your code compiles on your local machine
```

Comment:
Please replace with `Ensure each PR commit compiles and all unit and e2e tests pass your local machine.`

Reply:
Is it better if I say `Ensure each PR commit compiles and all unit and e2e tests pass on your local machine`.

Reply:
Yes, I missed a `on`